src/esp/gfx/RenderCamera.h

```diff
@@ -43,17 +41,10 @@ class RenderCamera : public Magnum::SceneGraph::AbstractFeature3D {
                            float zfar,
                            float hfov);
 
-  mat4f getProjectionMatrix();
-
-  mat4f getCameraMatrix();
-
-  MagnumCamera& getMagnumCamera();
-
-  void draw(MagnumDrawableGroup& drawables);
+  // expose overload for directly setting projection matrix
+  using MagnumCamera::setProjectionMatrix;
```

Comment:
I understand that "if a derived class redefines base class member method (e.g., setProjectionMatrix here) then all the base class methods with same name become hidden in derived class".

However, we can still call that method in base class using: `derivedObject.::BaseClassName.projectionMatrix(...)`
So I do not think we need it here.